### ISSUE TITLE
Cannot open clock interface

### ISSUE THREAD

**Issue by WACriminalG2** [Nuchaz/BiblioCraft#604]
When I sneak-click on the clock, my hand does the animation like I'm activating it, but nothing happens.

Are there any known mod conflicts that cause this?

**Comment by raindropworks** [comment]
The clock has an interface?  I can't get it to do anything either

**Comment by WACriminalG2** [comment]
Yup, per the wiki: https://www.bibliocraftmod.com/wiki/clock/

You can make it chime and supply redstone signals at times of your choosing.

**Comment by raindropworks** [comment]
Yeah, I can't get the interface to open.  I also don't know how to get the grandfather clock variant either.  The config file is default, but overall I'm seeming to have an issue accessing any GUI that isn't just a straight inventory

**Comment by psychofad** [comment]
make sure you don't have anything else assigned to the right click key, even shift+right click keybinds can cause issues, CarryOn mod uses shift+right click and can cause all kind of issues like that, with how often you actually have to move something with CarryOn, on our server we have re-assigned the keybind to the tilde key ( ` ), but you may find one you like better.

Also, to be clear, the correct key combination to open the clock interface is Sneak+Right Click by default

**Comment by psychofad** [comment]
@minakitty to make the Grandfather variant:
`To attach 2 clocks together and make a grandfather clock, place a clock on top of another clock in the same orientation. Then, using the screw gun or the hand drill, sneak right click one of the clocks, then sneak right click the second clock and they will connect.`

information is [located here](https://www.bibliocraftmod.com/wiki/clock/)
